pkgs/development/libraries/v8/8_x.nix

```diff
@@ -148,6 +148,7 @@ stdenv.mkDerivation rec {
     install -D d8 $out/bin/d8
     install -D -m644 obj/libv8_monolith.a $out/lib/libv8.a
     install -D -m644 icudtl.dat $out/share/v8/icudtl.dat
+    ln -s libv8.a $out/lib/libv8_monolith.a
```

Comment:
This should be copied instead, otherwise we have a reference to source

Reply:
Why a symbolic link would make a reference to source? I tried to figure out. But I still don't get it.

Reply:
🤔 the file is only in the build directory, so the symlink should be rewritten. False alarm